docs/topics/auth/default.txt

```diff
@@ -945,7 +945,7 @@ All authentication views
 This is a list with all the views ``django.contrib.auth`` provides. For
 implementation details see :ref:`using-the-views`.
 
-.. function:: login(request, template_name=`registration/login.html`, redirect_field_name='next', authentication_form=AuthenticationForm, extra_context=None, redirect_authenticated_user=False)
+.. function:: login(request, next_page=None, template_name=`registration/login.html`, redirect_field_name='next', authentication_form=AuthenticationForm, extra_context=None, redirect_authenticated_user=False)
```

Comment:
The login function takes `**kwargs`, and forwards it to `LoginView`, so without changing the function, we get the extra functionality.
Also, I mirrored the logout function's docs ([line 1100](https://github.com/ThinkChaos/django/blob/4034a5cbef7e9b6afb2f151ec2201ab5afbeeaf3/docs/topics/auth/default.txt#L1100)), do you still want me to change this?